api/controllers/game/editmode.js

```diff
@@ -0,0 +1,26 @@
+module.exports = function (req, res) {
+  const { isRanked } = req.body;
+  const promiseGame = gameService.findGame({ gameId: req.session.game });
+  const promisePlayer = userService.findUser({ userId: req.session.usr });
+  Promise.all([promiseGame, promisePlayer])
+    .then(function changeAndSave(values) {
+      const [game, player] = values;
+
+      const gameUpdates = {isRanked};
+
+      Game.publish([game.id], {
+        change:'editmode',
+        userId: player.id,
+        pNum: player.pNum,
```

Comment:
These can be grabbed from the session
```suggestion
        userId: req.session.usr,
        pNum: req.session.pNum,
```